lib/pages/login/login_page.dart

```diff
@@ -0,0 +1,128 @@
+import 'package:flutter/material.dart';
+import 'package:intl_phone_field/intl_phone_field.dart';
+
+class LoginPage extends StatefulWidget {
+  static const String routeName = '/loginpage';
+
+  const LoginPage({super.key});
+
+  @override
+  State<LoginPage> createState() => _LoginPageState();
+}
+
+class _LoginPageState extends State<LoginPage> {
+  String dropdownvalue = 'GDSC';
+
+  // List of items in our dropdown menu
+  var items = [
+    'GDSC',
+    'Eco Club',
+    'Item 3',
+    'Item 4',
+  ];
+
+  @override
+  Widget build(BuildContext context) {
+    double myWidth = MediaQuery.of(context).size.width;
```

Comment:
Try keeping the name standard instead of using my prefix

`Size size = MediaQuery.of(context).size;
double width = size.width;
double height = size.height;`